### ISSUE TITLE
MoBiToolboxForMatlab needed?

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/Reporting-Engine#270]
```
Warning: Name is nonexistent or not a directory: C:\Users\PavelBalazki\AppData\Roaming\Open Systems
Pharmacology\MoBiToolboxForMatlab 
```

Is it necessary to have the MoBiToolboxForMatlab to run the reporting engine? if so - a better description in the documentation should be provided. The current release does not come the with MatlabToolbox, and it should be pointed out where to download it (and how to install? Why does it look for it under "AppData\Roaming\Open Systems
Pharmacology"?)

**Comment by msevestre** [comment]
Yes it is needed and it won't be further developed as we are moving completely to R

**Comment by PavelBal** [comment]
Maybe add a note to the current release? Because the release does not come with the Matlab Toolbox, but the documentation of Qualification part does not mention that the toolbox has to be downloaded extra.
